index.js

```diff
@@ -67,12 +74,12 @@ function init() {
 
   updateSpecFromPackage();
   spec = patchSpec(predefinedSpec);
-  app.use('/api-spec', (req, res, next) => {
+  app.use(packageInfo.baseUrlPath +'/api-spec', (req, res, next) => {
```

Comment:
How about adding a test for it?

Minor: space is missing here and below

Reply:
fixed